packages/contracts/contracts/core/plugin/IMembership.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import {IDAO} from "../IDAO.sol";
+
+/// @notice An interface to be used by DAO plugins that define membership by announcing and renouncing individual addresses.
+interface IMembership {
+    /// @notice Emitted to announce the membership of new members.
+    /// @param members The list of new members.
+    event MembersAnnounced(address[] members);
+
+    /// @notice Emitted to renounce the membership of existing members.
+    /// @param members The list of exisiting members.
```

Comment:
Shouldn't it say `The list of **exiting** members`?
I assume we emit here the addresses that are no longer members

Reply:
It should be `existing`. Fixed in 9730bff961c0b41c2bc0fb189335b57bb7f61dac